### ISSUE TITLE
Facial AR Remote compatible Device

### ISSUE THREAD

**Issue by gn00410660** [Unity-Technologies/facial-ar-remote#27]
We are actually using an iPhone XR for testing, and it is showing that the device is not compatible, is an iPhone X strictly required for using the script? We did, however try it with an iPhone X, and it worked.

**Comment by mtschoen-unity** [comment]
The app should check whether ARKit face tracking is supported, rather than the model of the phone. This check might be failing for an iPhone XR. I don't have immediate access to one of these devices, but I will try to track down a solution.

**Comment by Janooba** [comment]
Using an iPhone XS gave us this issue as well. I had to manually change the code that checks for compatibility. It's currently set to only allow the base iPhone X, rather than > iPhoneX

**Comment by mtschoen-unity** [comment]
That's unfortunate. What code are you referring to? The intent is to use the ARKit plugin to identify whether the phone supports face tracking. @Janooba were you able to get it working? If so, could your post your fix?

**Comment by Janooba** [comment]
I just realized I was running code from the latest release, not the up-to-date master. The old code I had to fix was still checking the device generation in ClientGUI.cs in Awake().
I simply changed
```cs
if (Device.generation == DeviceGeneration.iPhoneX)
```
to
```cs
if (Device.generation >= DeviceGeneration.iPhoneX)
```
as a quick and dirty fix. (Line 64, ClientGUI.cs)

I see now that (as you stated earlier in this thread) the validation method has changed in the latest commits. My bad! I will update and reply if I still have issues.
